dist/tools/pic32prog/doc.md

```diff
@@ -0,0 +1,136 @@
+Flashing on Linux using pickit-3
+================================
+
+Allow flashing a pic32-wifire from linux using https://github.com/sergev/pic32prog
+
+
+Informations come from this comment
+
+https://github.com/RIOT-OS/RIOT/pull/6092#issuecomment-261987955
+
+
+Steps
+-----
+
+### Setup your computer
+
+I ran this on Ubuntu 16.04
+
+* Install Virtualbox
+* Add yourself to the 'vboxusers' group
+  `sudo usermod -a -G vboxusers your_username`
+* Reboot (was necessary for the ext-pack may be possible without)
+
+### Setup Windows VM
+
+You can download a free 90 days valid windows virtualbox image from:
```

Comment:
Why do I need to download it?

Reply:
That's only if you need to change the firmware of your pickit3 (usually you'll need), and it can only be done under windows.

Reply:
*I* know this, but it isn't clear from this document.

Reply:
I could add a global summary at the beginning explaining what should be done and how:

* Update pickit-3 firmware to allow flashing from linux
* Updating the firmware should be done from a windows computer
   * This tutorial will explain how to do it from a virtual machine.

Reply:
I do net think it would then be necessary to repeat it after if explained at the beginning.